src/gobimportclient/import_client.py

```diff
@@ -142,15 +142,13 @@ def publish(self):
         publish("gob.workflow.proposal", "fullimport.proposal", import_message)
 
     def start_import_process(self):
-        self.connect()
-        self.read()
-        self.enrich()
-        self.convert()
-        self.validate()
-        self.publish()
-        '''
         try:
-
+            self.connect()
+            self.read()
+            self.enrich()
+            self.convert()
+            self.validate()
+            self.publish()
         except Exception as e:
             self.log('error', f'{e}')
```

Comment:
Shouldn't you include import failed in the error message?

Reply:
Import failed has been added